attest/attest.go

```diff
@@ -145,6 +146,14 @@ func (k *AK) AttestationParameters() AttestationParameters {
 	return k.ak.attestationParameters()
 }
 
+// Certify uses the attestation key to certify the key with `handle`. It returns
+// certification parameters which allow to verify the properties of the attested
+// key. Depending on the actual instantiation it can accept different handle
+// types (e.g., tpmutil.Handle or uintptr).
+func (k *AK) Certify(tb tpmBase, handle interface{}) (*CertificationParameters, error) {
```

Comment:
This probably shouldn't be an interface? Where does the handle come from? Is it from the same process? Can we expose a "type Handle struct" with internal fields?

Reply:
handle will come from this package (tpmutil.Handle) or from Windows' certtostore (uintrptr).
You mean something like this?
<pre>
type Handle struct {
  Native tpmutil.Handle
  PCP uintrptr
}
</pre>